
---
int64 act
float64 hmin_green
float64 hmax_green
float64 smin_green
float64 smax_green
float64 vmin_green
float64 vmax_green
float64 hmin_obs
float64 hmax_obs
float64 smin_obs
float64 smax_obs
float64 vmin_obs
float64 vmax_obs
float64 hmin_red1
float64 hmax_red1
float64 smin_red1
float64 smax_red1
float64 vmin_red1
float64 vmax_red1
float64 hmin_red2
float64 hmax_red2
float64 smin_red2
float64 smax_red2
float64 vmin_red2
float64 vmax_red2
float64 arprom_green
float64 arprom_red
float64 arprom_obs
int64 xmin1
int64 xmin2
int64 ymin1
int64 ymin2
int64 xmax1
int64 xmax2
int64 ymax1
int64 ymax2
